images/build/debian-iptables/distroless/get-binaries-from-package.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+# Copyright 2022 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# USAGE: get-binaries-from-package.sh /opt/stage package1 package2
+# TODO: The above is not true. I will fix it!
+#
+# Stages all the packages and its dependencies (+ libraries and copyrights) to $1
+#
+# This is intended to be used in a multi-stage docker build with a distroless/base
+# or distroless/cc image.
+
+. package-utils.sh || exit 1 
+
+get_file_list() {
```

Comment:
that name suggests it's just returning a list of files, not actually staging things

Reply:
changed to `stage_file_list`